cmd/chisel/helpers.go

```diff
@@ -0,0 +1,70 @@
+package main
```

Comment:
I am not sure that a helpers file is the best place for these couple of new functions. Maybe it makes more sense to move them to the `setup` package where the rest of the `release` functions are located. I am not sure about that either, what do you think?

Reply:
It makes sense to move them to `setup` since there are similar functions there. But these functions will probably not be used in the `setup` package. At least currently, I see usage only from the `main` package. Additionally, moving to `setup` means exporting the names and having public exposure. I am not sure if it is a good idea to export them.

Reply:
If these functions are only going to be used for chisel commands, if they are more general than that we could move them even if they had only one usage. We already do something similar for things like the `jsonwall` package that is only going to be used in a single place for now at least. 

Else, we can keep it in the helpers file and move them in the future if needed.